htdocs/compta/prelevement/card.php

```diff
@@ -188,7 +190,7 @@
 llxHeader('', $langs->trans("WithdrawalsReceipts"));
 
 if ($id > 0 || $ref) {
-	$head = prelevement_prepare_head($object);
+	$head = prelevement_prepare_head($object, $salaryBonPl);
```

Comment:
There is no need to have ad added parameter here.
The information of the type of BonPrelevement is already stored into the BonPrelevement object

Reply:
i added for check if invoice is salary and change onglet Bills to Salaries 
because I used the method above in line 75
$salaryBonPl = $object->checkIfSalaryBonPrelevement($id);